torch/__init__.py

```diff
@@ -186,6 +186,7 @@ def set_default_dtype(d):
 from .tensor import Tensor
 from .storage import _StorageBase
 
+tensor = _C.tensor  # remove duplicate name, and fix to shortcut function
```

Comment:
It can move to below line of `torch/__init__.py#L250`
```
for name in dir(_C._VariableFunctions):
    globals()[name] = getattr(_C._VariableFunctions, name)
```